pkg/crane/crane_test.go

```diff
@@ -288,7 +328,7 @@ func TestCraneFilesystem(t *testing.T) {
 
 func TestBadInputs(t *testing.T) {
 	t.Parallel()
-	invalid := "@@@@@@"
+	invalid := "/dev/null/@@@@@@"
```

Comment:
Is `@@@@@@` no longer invalid?

Reply:
This makes it both an invalid file to read and an invalid file to write.